bigquery/google/cloud/bigquery/job.py

```diff
@@ -1316,58 +1315,43 @@ class QueryJobConfig(object):
     server defaults.
     """
 
-    _QUERY_PARAMETERS_KEY = 'queryParameters'
-    _UDF_RESOURCES_KEY = 'userDefinedFunctionResources'
-
     def __init__(self):
         self._properties = {}
 
     def to_api_repr(self):
-        """Build an API representation of the copy job config.
+        """Build an API representation of the query job config.
 
-        :rtype: dict
-        :returns: A dictionary in the format used by the BigQuery API.
+        Returns:
+            dict: A dictionary in the format used by the BigQuery API.
         """
         resource = copy.deepcopy(self._properties)
 
         # Query parameters have an addition property associated with them
         # to indicate if the query is using named or positional parameters.
-        query_parameters = resource.get(self._QUERY_PARAMETERS_KEY)
+        query_parameters = resource.get('queryParameters')
         if query_parameters:
-            if query_parameters[0].name is None:
+            if query_parameters[0].get('name') is None:
                 resource['parameterMode'] = 'POSITIONAL'
             else:
                 resource['parameterMode'] = 'NAMED'
 
-        for prop, convert in self._NESTED_PROPERTIES.items():
-            _, to_resource = convert
-            nested_resource = resource.get(prop)
-            if nested_resource is not None:
-                resource[prop] = to_resource(nested_resource)
-
         return resource
 
     @classmethod
     def from_api_repr(cls, resource):
         """Factory: construct a job configuration given its API representation
 
-        :type resource: dict
-        :param resource:
-            An extract job configuration in the same representation as is
-            returned from the API.
+        Args:
+            resource (dict): A query job configuration in the same
+                representation as is returned from the API.
 
-        :rtype: :class:`google.cloud.bigquery.job.QueryJobConfig`
-        :returns: Configuration parsed from ``resource``.
+        Returns:
+            :class:`~google.cloud.bigquery.job.QueryJobConfig`: Configuration
```

Comment:
I think the `:class:` piece isn't necessary with Google-style. Just `google.cloud.bigquery.job.QueryJobConfig: Config...`